test/remote/password_change_tests.js

```diff
@@ -95,6 +95,36 @@ TestServer.start(config)
   )
 
   test(
+    'change password on locked account',
+    function (t) {
+      var email = server.uniqueEmail()
+      var password = 'wibble'
+      var client
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+        .then(
```

Comment:
inconsistent style, there is sometimes 2 space before `.then` sometimes not.
above you got:
![image](https://cloud.githubusercontent.com/assets/128755/9092162/5bedd682-3b72-11e5-9141-f23f7cad20f6.png)